src/site/topic25.rst

```diff
@@ -108,6 +108,130 @@ Bogosort
 Bubble Sort
 ===========
 
+.. image:: img/sort_bubblesort.png
+   :width: 500 px
+   :align: center
+   :target: https://en.wikipedia.org/wiki/Bubble_sort
```

Comment:
Gif is a bit slow imo. If you choose not to make your own, you may want to download (and cite) the image, since wikipedia may remove it at any time.